spec/seeds/seeds_chitter.sql

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Nice way of inserting values into tables concisely, I repeat a lot of code in mine